src/util.ts

```diff
@@ -54,3 +58,40 @@ export function generateTableChoices<T>(
 
   return choicesOptions;
 }
+
+/**
+ * Function to build a help section for command help.
+ * Takes a string to be used as section header text and an array of enums
+ * that identify the variable or property to be included in the help
+ * body.
+ *
+ * @param header
+ * @param vars
+ */
+export function toHelpSection(
+  header: string,
+  ...vars: Array<OrgConfigProperties | SfdxPropertyKeys | EnvironmentVariable>
+): HelpSection {
+  const body = vars
+    .map((v) => {
+      const orgConfig = ORG_CONFIG_ALLOWED_PROPERTIES.find(({ key }) => {
+        return key === v;
+      });
+      if (orgConfig) {
+        return { name: orgConfig.key, description: orgConfig.description };
+      }
+      const sfdxProperty = SFDX_ALLOWED_PROPERTIES.find(({ key }) => key === v);
```

Comment:
Instead of referencing `SFDX_ALLOWED_PROPERTIES` and `OrgConfigProperties` separately, you should be able to just reference `Config.allowedProperties` which contains both: https://github.com/forcedotcom/sfdx-core/blob/v3/src/config/config.ts#L263

I think that would be preferable in the long term too since we won't have to update the list of allowed configs in two separate places